README.md

```diff
@@ -29,6 +29,11 @@ $ docker build --target development --tag todo-app:dev .
 $ docker run -d -p 5000:5000 --env-file ./.env todo-app:dev 
 ```
 
+To run the docker container with a bind mount so we can change the Python files which will directly reflect without rebuilding the image / rerunning the container , we need to run with the following command :
+```bash
+docker run --env-file ./.env -p 5000:5000 --mount type=bind,source="$(pwd)",target=/DevOps-Course-Starter todo-app:dev
```

Comment:
Might be worth adding a comment saying this only works with UNIX shells (it won't work in powershell for example)

Reply:
Added